### PR TITLE
Fixed errors Pattern matched are not exhaustive. Though I left parser…

### PR DESCRIPTION
**Pull request by frankjohnlee**
Fixed errors Pattern matched are not exhaustive. Though I left parser…

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,4 @@ cabal.sandbox.config
 
 # IDE project files
 .idea
+app/Svg/Generator.hs
```
Comment (david-yz-liu) [inline .gitignore]: @frankjohnlee please revert this change. This is not appropriate (either for this specific pull request, or in general).

File: app/Svg/Parser.hs [hunk 2]
```diff
@@ -362,6 +362,7 @@ updateShape fill r =
                               Hybrid   -> Hybrid
                               BoolNode -> BoolNode
                               Node     -> Node
+                              _ -> ""
```
Comment (david-yz-liu) [inline app/Svg/Parser.hs]: @frankjohnlee This is the right idea, but contains a type error. You can use `undefined` instead of `""` here.

Comment (frankjohnlee) [inline app/Svg/Parser.hs]: Thanks David. I have made this change.
